packages/lesswrong/components/sequences/LargeSequencesItem.tsx

```diff
@@ -123,53 +134,60 @@ export const LargeSequencesItem = ({sequence, showAuthor=false, classes}: {
   showAuthor?: boolean,
   classes: ClassesType,
 }) => {
-  const { UsersName, ContentStyles, SequencesSmallPostLink, ContentItemTruncated } = Components
+  const { UsersName, ContentStyles, SequencesSmallPostLink, ContentItemTruncated, LWTooltip } = Components
   
   const [expanded, setExpanded] = useState<boolean>(false)
 
   const cloudinaryCloudName = cloudinaryCloudNameSetting.get()
 
+  let posts : PostsList[] = []
+  sequence.chapters?.forEach(chapter => chapter.posts?.forEach(post => posts.push(post)))
+  const totalWordcount = posts.reduce((prev, curr) => prev + (curr?.contents?.wordCount || 0), 0)
+
+  const highlight = sequence.contents?.htmlHighlight || ""
```

Comment:
I prefer `??` to `||`, especially in cases where `0` or `''` are semantically meaningful values, to reduce confusion and possible sources of bugs.  (`??` will only coalesce for `null | undefined` on the left side, unlike `||`, which will also do it for `0`, `false`, `''`, etc.)